src/BinSkim.Rules/NativeInterop.cs

```diff
@@ -218,10 +218,15 @@ internal struct CRYPTOAPI_BLOB
         internal const string szOID_CERT_STRONG_KEY_OS_1 = "1.3.6.1.4.1.311.72.2.1";
 
         [DllImport("winterop.dll", EntryPoint = "HashPublicKeyInfo", CharSet = CharSet.Unicode, ExactSpelling = true, PreserveSig = false)]
-        internal static extern void HashPublicKeyInfo(IntPtr certContext, byte[] publicKeyInfoHashed, ref uint sizePublicKeyInfoHashed);
+        private static extern void HashPublicKeyInfo(IntPtr certContext, byte[] publicKeyInfoHashed, ref uint sizePublicKeyInfoHashed);
+
+        internal static void HashPublicKeyInfoWrapper(IntPtr certContext, byte[] publicKeyInfoHashed, ref uint sizePublicKeyInfoHashed)
```

Comment:
Do these wrappers exist with this implementation because the interop helpers are present in code but currently unused? It's hard for me to tell what's going on here using the GitHub diff UI. :) I will look at this more closely after enlisting in your branch.

Reply:
We don't use these at the moment, but I think I must have just missed converting the wrappers.  Updated.